#!/usr/bin/python3
def raise_exception_msg(message=""):
    """ my Raise a message function """
    raise NameError(message)
